modules/tinymce/src/core/test/ts/browser/UndoManagerTest.ts

```diff
@@ -440,6 +440,35 @@ UnitTest.asynctest('browser.tinymce.core.UndoManager', function (success, failur
     ok(editor.isDirty(), 'Dirty state should be true');
   });
 
+  suite.asyncTest('Dirty state on second AddUndo', function (editor, done, die) {
```

Comment:
Arrow functions please :smile:

Reply:
You kids with your Hula Hoops, your compact cassettes, and your convenient lambda syntax... :)